tools/discord_notifier.py

```
@@ -0,0 +1,159 @@
+
+
+def format_story_message(packet: Dict[str, Any]) -> str:
+    """Format a story packet into a Discord message."""
+    title = packet.get("title_human") or packet.get("title_raw", "Untitled")
+    pr_url = packet.get("links", {}).get("pr_url", "")
+    why = packet.get("why", "")
+    highlights = packet.get("highlights", [])
+    
+    lines = [
+        f"📹 **Time to record** — {title}",
+        f"<@379607746510848000>",  # Tag @paulchrisluke
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)